Packs/PAN-OS/Integrations/Panorama/README.md

```diff
@@ -8985,3 +8985,34 @@ Returns a list of all device groups from Panorama.
 | **Path** | **Type** | **Description** |
 | --- | --- | --- |
 | Panorama.DeviceGroupNames | string | The list of device groups. |
+
+### pan-os-export-tech-support-file
+
+***
+Exports a tech support file (TSF).
+
+#### Base Command
+
+`pan-os-export-tech-support-file`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| interval_in_seconds | The polling interval (in seconds). Default is 60. | Optional | 
+| timeout | The polling timeout (in seconds). Default is 1200. | Optional | 
+| job_id | The job ID to use when polling. | Optional | 
```

Comment:
```suggestion
```

this is not relevant for the customers, only for our internal use

Reply:
@ali-sawyer please re-run the SDK's `generate-docs` command following the YAML changes (d8c73d1e59ccdd451c8e0d8edfcf1ebc43ecf6ce).